signals/src/signal.rs

```diff
@@ -925,4 +1036,27 @@ mod tests {
         assert_eq!(super::Signal::poll(&mut s), super::State::Changed(15));
         assert_eq!(super::Signal::poll(&mut s), super::State::NotChanged);
     }
+
+    #[test]
+    fn test_broadcast() {
+        use super::Signal;
+
+        let (s, r) = super::unsync::mutable(0);
+
+        let mut r1 = r.broadcast();
+        let mut r2 = r1.clone();
+
+        // TODO: NotChanged checks panic due to futures::task::current()
+
+        assert_eq! (r1.poll(), super::State::Changed(0));
```

Comment:
Nit: no space after `(`

Reply:
Eurgh our c++ style at work is mandated spaces. I try my best to catch them from leaking outside of the office...